task_sdk/tests/execution_time/test_supervisor.py

```diff
@@ -349,11 +349,14 @@ def test_state_conflict_on_heartbeat(self, captured_logs, monkeypatch, mocker):
         """
         import airflow.sdk.execution_time.supervisor
 
-        monkeypatch.setattr(airflow.sdk.execution_time.supervisor, "MIN_HEARTBEAT_INTERVAL", 0.1)
+        # Heartbeat every time around the loop
+        monkeypatch.setattr(airflow.sdk.execution_time.supervisor, "MIN_HEARTBEAT_INTERVAL", 0.0)
```

Comment:
This setting caused it to hit three requests every time before the fix.

Reply:
huh 👍